airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/model_to_component_factory.py

```diff
@@ -435,7 +435,10 @@ def _get_class_from_fully_qualified_class_name(class_name: str) -> Any:
         split = class_name.split(".")
         module = ".".join(split[:-1])
         class_name = split[-1]
-        return getattr(importlib.import_module(module), class_name)
+        try:
+            return getattr(importlib.import_module(module), class_name)
+        except AttributeError:
+            raise ValueError(f"Class {class_name} could not be found.")
```

Comment:
Should this include the fully qualified class name instead of just the single class name at the end?

For example, if in one stream I point to `source_mysource.events.CustomPaginationStrategy`, and in another stream I point to  `source_mysource.users.CustomPaginationStrategy`, it would be useful to know which one could not be found.

This may be a contrived example that could be solved by just using better class names, but what do you think?

Reply:
good callout. I updated so the fully qualified class name is shown:
`ValueError: Could not load class unit_tests.sources.declarative.parsers.testing_components.NonExistingClass`